internal/backend/deployments/deployments.go

```diff
@@ -53,25 +53,25 @@ func New(l *zap.Logger, cfg *Config, db db.DB, telemetry *telemetry.Telemetry) s
 }
 
 func (d *deployments) Autodrain() {
-	d.l.
-		With(zap.Duration("interval", d.cfg.AutoDrainingDeactivationInterval),
-			zap.Duration("jitter", d.cfg.AutoDrainingDeactivationJitter)).
-		Info("periodically autoinactivating drained deployments")
+	l := d.l.With(
+		zap.Duration("auto_drain_interval", d.cfg.AutoDrainingDeactivationInterval),
+		zap.Duration("auto_drain_jitter", d.cfg.AutoDrainingDeactivationJitter),
+	)
+	l.Info("periodically auto-deactivating drained deployments")
 
 	for {
-		t := d.cfg.AutoDrainingDeactivationInterval + time.Duration(rand.Float32()*float32(d.cfg.AutoDrainingDeactivationJitter))
-
-		d.l.Debug("waiting before next autodeactivation", zap.Duration("duration", t))
```

Comment:
it's debug... but oh well

Reply:
"Time is passing" is a category of log messages that not only don't add useful info, they dilute the value of logging because the extra noise makes it harder to find/browse/group useful info, not to mention they cost much more.